session-keys-local-storage/src/app/page.tsx

```diff
@@ -23,12 +24,14 @@ import chain from "@/config/chain";
  */
 export default function Home() {
   const { address, isConnected } = useAccount();
+  const { disconnect } = useDisconnect();
```

Comment:
I think this should use the logout function from the AGW hook useLoginWithAbstract